edx_proctoring/templates/practice_exam/submitted.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     {% blocktrans %}
-      Practice exams do not affect your grade or your credit eligibility.
+      Practice exams do not affect your grade.
```

Comment:
I created https://openedx.atlassian.net/browse/EDUCATOR-403 for differentiating text based on credit/MicroMaster's.